ibr/ibr-sdk/package.json

```diff
@@ -14,17 +14,30 @@
     "test": "ava"
   },
   "dependencies": {
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "earcut": "^2.2.2",
     "pbf": "^3.2.1",
+    "react": "^16.13.1",
+    "react-dom": "^16.8.0",
```

Comment:
can we use react in the example.html but not in the sdk? requiring developers to use react might be too opinionated

i.e. our example demo can be a react component, but we should only export THREE.js wrapped functions instead of exporting react components and forcing users to use react